tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -142,45 +144,45 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 				Name:    "basic-v4",
 			},
 		}
-
+		
 		for _, clusterCfg := range clusterCfgs {
 			localCfg := clusterCfg
 			ginkgo.It(fmt.Sprintf("[TiDB Version: %s] %s", localCfg.Version, localCfg.Name), func() {
 				cluster := newTidbCluster(ns, localCfg.Name, localCfg.Version)
 				cluster.Spec.EnablePVReclaim = pointer.BoolPtr(true)
 				// support reclaim pv when scale in tikv or pd component
 
-				tests.CreateTidbClusterOrDie(cli, cluster)
+				crdUtil.CreateTidbClusterOrDie(cluster)
```

Comment:
in the future, can we remove these wrappers? in ginkgo e2e, we don't need to call slack.

Reply:
I think we can add a switch to control it.

Reply:
we can remove thin wrappers such as`CreateTidbClusterOrDie` which just call methods on kubernetes client